### PR TITLE
added stuff

### PR DESCRIPTION
**Pull request by simes900**
added summary screen for item when tapped on and added padding to add screen .

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -10,10 +10,7 @@
         android:supportsRtl="true"
         android:theme="@style/Theme.Pomo">
         <activity android:name=".activities.PomodoroDetails"></activity>
-        <activity
-            android:name=".MainActivity2"
```
Comment (kkuang70) [inline app/src/main/AndroidManifest.xml:14]: Name this better

File: app/src/main/java/com/example/pomo/activities/ItemActivity.kt [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+
+
+class ItemActivity:AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_item_information)
+        val item = intent.getSerializableExtra("POMOITEM") as PomodoroSession
+       // Toast.makeText(this, item.toString(), Toast.LENGTH_SHORT).show()
```
Comment (kkuang70) [inline app/src/main/java/com/example/pomo/activities/ItemActivity.kt]:
```suggestion
```

File: app/src/main/java/com/example/pomo/activities/MainActivity.kt [hunk 3]
```diff
@@ -31,8 +34,16 @@ class MainActivity : AppCompatActivity(), PomodoroAdapter.OnItemClickListener {
     }
 
     override fun onItemClick(position: Int) {
-        Toast.makeText(this, "Item $position clicked", Toast.LENGTH_SHORT).show()
+        val pomo = adapter.getData(position)
+        //Toast.makeText(this, "Item ${adapter.getData(position)} clicked", Toast.LENGTH_SHORT).show()
```
Comment (kkuang70) [inline app/src/main/java/com/example/pomo/activities/MainActivity.kt]:
```suggestion
```

File: app/src/main/java/com/example/pomo/activities/ItemActivity.kt (hunk 2 above)
Comment (simes900) [reply app/src/main/java/com/example/pomo/activities/ItemActivity.kt]: this toast was for testing purposes to display object to be passed to next activity

File: app/src/main/java/com/example/pomo/activities/ItemActivity.kt [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+data class PomodoroSession (
+    @PrimaryKey(autoGenerate = true)
+    val id: Int,
+    val title: String,
+    val description: String,
+    val studyMinutes: Float,
+    val breakMinutes: Float,
+    val startDateTime: String,
+    val endDateTime: String
+) : Serializable*/
```
Comment (kkuang70) [inline app/src/main/java/com/example/pomo/activities/ItemActivity.kt]: Remove code no longer used

File: app/src/main/java/com/example/pomo/activities/MainActivity.kt [hunk 5]
```diff
@@ -31,8 +34,16 @@ class MainActivity : AppCompatActivity(), PomodoroAdapter.OnItemClickListener {
...
+        val i =Intent(this, ItemActivity::class.java)
+        i.putExtra("POMOITEM", l1 as Serializable )
+        startActivity(i)
 /*
+
+val intent = Intent(this, PomodoroDetails::class.java).apply {
+            putExtra("Message", pomodoroDataItem)
         mPomodoroSessionViewModel.deletePomodoroSession(adapter.getData(position))
         adapter.notifyItemRemoved(position)*/
     }
```
Comment (kkuang70) [inline app/src/main/java/com/example/pomo/activities/MainActivity.kt]: Remove code no longer used

**Review by kkuang70 — CHANGES_REQUESTED** [review]

**Review by kkuang70 — APPROVED** [review]

**Pull request merged**